atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -386,12 +387,16 @@ default String type() {
         return TYPE;
     }
 
-    @Override
     @Value.Default
     default int concurrentGetRangesThreadPoolSize() {
         return poolSize() * servers().numberOfThriftHosts();
     }
```

Comment:
might be worth adding a comment here: if this is ever changed so that it defaults to more than zero even when servers() is empty, changes may need to be made in the reloadable KVS

Reply:
Good point. This will be made optional once the 3rd PR is in, but I'll leave a comment to note incase there are changes in between.